[esx_addons]/esx_lscustom/server/main.lua

```diff
@@ -33,6 +34,20 @@ AddEventHandler('esx_lscustom:buyMod', function(price)
 	end
 end)
 
+RegisterNetEvent('esx_lscustom:changeVehicleData')
+AddEventHandler('esx_lscustom:changeVehicleData', function(type, data)
+	local source = source
+	if type == "add" then
+		VehicleData[source] = {plate = data.plate, props = data.props}
```

Comment:
You can replace this line with :

```lua
VehicleData[source] = data
```

Reply:
yes this is true
we can get plate from vehicle's prop
but it's very easy for find vehicle in tables